src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/OperationController.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi.Controllers
+{
+    [Route(ControllerName)]
+    [ApiController]
+    [HostRestriction]
+    [Authorize(Policy = AuthConstants.PolicyName)]
+    public class OperationController : ControllerBase
+    {
+        private readonly ILogger<OperationController> _logger;
+        private readonly EgressOperationStore _operationStore;
+
+        public const string ControllerName = "operation";
```

Comment:
nit: Should this be "operations" since it represents the entire collection of operations?

Reply:
You resolved this comment, but I don't see any action or follow up on it. Are you planning on making this change?

Reply:
I think I misunderstood the scope of your comment: you want to rename the entire controller not just the field.

Reply:
Correct, I would expect the routes to be `/operations` and `/operations/{id}` to insinuate that I'm listing all of the operations and that I'm getting a single operation by id from the collection of operations, respectively.